commands/addons/create.js

```diff
@@ -46,16 +52,47 @@ function * run (context, heroku) {
         'x-heroku-legacy-provider-messages': 'true'
       }
     })
+
     cli.action.done(cli.color.green(util.formatPrice(addon.plan.price)))
   }))
-  if (addon.config_vars.length) {
-    let configVars = addon.config_vars.map(c => cli.color.configVar(c)).join(', ')
-    cli.log(`Created ${cli.color.addon(addon.name)} as ${configVars}`)
-  } else {
-    cli.log(`Created ${cli.color.addon(addon.name)}`)
+
+  if (!addon.config_vars || !addon.config_vars.length) { addon.config_vars = [] }
```

Comment:
The problem I ran into is that the design dictates different output for async `--wait` or not.